spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -48,6 +48,32 @@
       expect(response).to redirect_to hub_assigned_clients_path
     end
 
+    render_views
```

Comment:
Why `render_views`? I don't see any assertions on the content of the page in this test. For me, since I know `render_views` is something to be used sparingly, I check that the code I submit for review only uses it when needed as part of reading the diff before submitting the review.